Kros.SourceGenerators.PropertyAccessorsGenerator/AnalyzerReleases.Unshipped.md

```diff
@@ -0,0 +1,8 @@
+﻿; Unshipped analyzer release
+; https://github.com/dotnet/roslyn-analyzers/blob/main/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md
+
+### New Rules
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+KROS001 | Kros.SourceGenerators.PropertyAccessors | Warning | GeneratorExecutionContextExtensions
```

Comment:
Idčko by malo byť unikátne, takto sa môže stať, že v ďalšom generátore použijeme rovnaké. Preto by som dal `KRPA001`, ... taktiež to zmeniť dole v kóde, kde sa to používa.